src/common/components/hosts/EditHostForm.tsx

```diff
@@ -56,6 +57,7 @@ const EditHostForm = ({
   getEditErrorMessage,
 }: EditHostFormProps) => {
   const hostnameInputRef = React.useRef<HTMLInputElement>();
+  const [isEditInProgress, setIsEditInProgress] = React.useState<boolean>(false);
```

Comment:
Could we rename to `isSaveInProgress`, `setIsSaveInProgress`? With the _edit_ word I think "the user is editing the form" but in this case it seems to be "the user's changes being applied" for which the word _save_ seems more fitting.

Reply:
Makes sense, I'll rename it